bioio_tifffile/reader.py

```diff
@@ -153,7 +153,9 @@ def scenes(self) -> typing.Tuple[str, ...]:
     @property
     def physical_pixel_sizes(self) -> types.PhysicalPixelSizes:
         """Return the physical pixel sizes of the image."""
-        if self._physical_pixel_sizes is None:
+        if self._physical_pixel_sizes is None or any(
```

Comment:
Does seem like we would hit this a lot of someone were to ask for it often when we can't actually get access to the pixel sizes, but that seems like an unlikely case and I see why you'd want to do this so LGTM